.github/workflows/maintenance-cherry-pick-workflow.yml

```diff
@@ -0,0 +1,40 @@
+name: cherry-pick on maintenance branch
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    outputs:
+      output: ${{ steps.cherry_pick.outputs.output }}
+    if: |
+      github.event.pull_request.merged == true &&
+      contains(github.event.pull_request.labels.*.name, 'target: maintenance')
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Cherry-pick changes into maintenance"
+      id: cherry_pick
+      run: |
+        git checkout maintenance
```

Comment:
Maybe this is more a general question about maintenance branch naming.
I'd prefer for each (current and new) major version to create a dedicated branch (e.g. `13.x` and `14.x`) and to update the target branch when creating the first major release.
@jeripeierSBB, @mhaertwig Thoughts?

Reply:
Should we maybe have corresponding labels so it would be transparent where it would be cherry-picked to?
e.g. `target: 13.x` cherry-picks to `13.x` and `target: 14.x` cherry-picks to `14.x` and if both labels are set, it cherry-picks to both? The auto-label would only assign the manually configured label and we can switch it, once we want to move to the next major version.
But I'm open to discussion, if this seems to complex.